src/lightning_lite/strategies/launchers/multiprocessing.py

```diff
@@ -121,7 +121,7 @@ def _wrapping_function(
     ) -> None:
         if global_states:
             global_states.restore()
-        self._strategy._local_rank = process_idx
+        os.environ["LOCAL_RANK"] = str(process_idx)
```

Comment:
This is now the same way the subprocess launcher sets the local rank for the ddp strategy.